src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -4,7 +4,7 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
   python:
-    package_name: library
+    package_name: google.example.library.v1
```

Comment:
I agree with making this more realistic. BTW, the actual path would be something like `google.cloud.gapic.library.v1`.